layouts/shortcodes/featured/related-stories.html

```diff
@@ -0,0 +1,200 @@
+{{ $side := .Get "side" | default "left" }}
+{{ $limit := .Get "limit" | default 4 | int }}
+{{ $sideClass :=
+  cond (eq $side "right")
+  "float-right sm:px-4 lg:-mr-24 xl:-mr-40"
+  "float-left sm:px-6 lg:-ml-24"
+}}
+{{ $self := .Page.RelPermalink }}
+{{ $params := partialCached "helper/page-params" .Page $self }}
+{{ $topic := $params.topic }}
+
+{{ $link := .Get "link" }}
+{{ $cta := .Get "cta" }}
+{{ $name := .Get "title" }}
+
+{{ with .Get "topic" }}
+  {{ $topic = (site.Taxonomies.topic.Get (urlize . )).Page }}
+  {{ $name = "Spotlight PA" }}
+{{ end }}
+{{ with .Get "title" }}
+  {{ $name = . }}
+{{ end }}
+{{ $pages := $topic.Pages.ByPublishDate }}
+{{ if gt $pages.Len 1 }}
+  {{ $jsOpts := dict
+    "minify" true
+  }}
+  {{ $js := resources.Get "js/esbuild/smoothscroll.js" | js.Build $jsOpts }}
+  <script type="module">
+    {{ $js.Content | safeJS }}
+  </script>
```

Comment:
If there’s more than one related block on the page, it will execute this twice, but if we change it to be src=$js.RelPermalink and keep type=module, it should only execute once even if it is included multiple times.